docs/keyboard.md

```diff
@@ -66,6 +66,8 @@ follows.
 | Ctrl + Alt + x, f, x | focusCharacterColumnInput |
 | Ctrl + Alt + x, f, y | focusCharacterRowInput |
 | Ctrl + Alt + x, f, w | focusWorldSelector |
+| Ctrl + Alt + x, s, l | swapCurrentStepWithPreviousStep |
```

Comment:
Maybe 'm' for "move" rather than 's'. As "move" is what we use in the UI announcements.